binr/rafind2/rafind2.c

```diff
@@ -5,7 +5,7 @@
 #include <getopt.c>
 
 #include <r_types.h>
-#include <r_print.h>
+#include <r_util/r_print.h>
```

Comment:
i think that this include should be inside r_util.h, so we can get rid of all those lines

Reply:
@radare Ok, I'll fix it. :-)